### DISCUSSION TITLE
I need help settling up MelonsDs on RetoArch

### DISCUSSION THREAD

**Discussion by itsangelajf** [rafaelvcaetano/melonDS-android#1308] [Support]
I have not been able to configure the WFC, I want to use the multiplayer option but Idk how to configure RetroArch

**Comment by rafaelvcaetano** [comment]
I do not develop the RetroArch core of melonDS. I do know that there are 2 versions. One called "melonDS" and another one called "melonDS DS". I suggest you use the later since it is being actively developed. If you need support from the developer, check out it's repository here: https://github.com/JesseTG/melonds-ds
